programs/programs/co/utility_bill_pay/calculator.py

```
@@ -1,7 +1,30 @@
+from integrations.services.sheets import GoogleSheetsCache
+from programs.co_county_zips import counties_from_screen
 from programs.programs.calc import Eligibility, ProgramCalculator
+import programs.programs.messages as messages
+
+
+class IncomeLimitsCache(GoogleSheetsCache):
+    sheet_id = "1ZzQYhULtiP61crj0pbPjhX62L1TnyAisLcr_dQXbbFg"
+    range_name = "A2:K"
+    default = {}
```

Ruff (0.13.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)